go/src/koding/kites/kloud/metrics/metrics.go

```diff
@@ -69,6 +73,12 @@ func (p *Publisher) Publish(r *kite.Request) (interface{}, error) {
 		return nil, err
 	}
 
+	go func() {
+		if _, err := http.Post(p.metricsEndpoint, "application/json", bytes.NewReader(r.Args.Raw)); err != nil {
```

Comment:
http.DefaultClient has no request timeout set and I think it should.

Also I think Publisher could have `Client *http.Client` field and default to http.DefaultClient if nil.